features/update/deduplication/feature.php

```diff
@@ -25,6 +25,7 @@ class FeatureWorkflow
     public function run()
     {
         yield Workflow::await(fn(): bool => $this->counter >= 2);
+        yield Workflow::await(fn(): bool => Workflow::allHandlersFinished());
```

Comment:
@Sushisource @roxblnfk LGTY?

Reply:
This is addressing the new server change to update errors: when a workflow completes and an update is in Accepted state, that used to be gRPC NOT FOUND, but is now an update failure. Without the change here, we are getting

```
AcceptedUpdateCompletedWorkflow: Workflow Update failed because the Workflow completed before the Update completed.
```

I have to admit I don't fully understand where the gRPC NOT FOUND was being handled, but the fix here seems like good practice and fixes the test.

Reply:
Yeah I think this change is fine for the purposes of this PR.

Reply:
I think we can just suppress this exception: https://github.com/temporalio/sdk-php/commit/0f0ceec6d77e3abdd3e7ca21744e283dc0563476

Reply:
Thanks @roxblnfk. So I guess both work -- I think I'm finding the wait `allHandlersFinished` a bit cleaner since it's something that we recommend. WDYT?

Reply:
What about `$this->counter >= 2 && Workflow::allHandlersFinished()`?
Looks like that counter is important

Reply:
@dandavison 
There should be a condition `$this->counter >= 2 && Workflow::allHandlersFinished()` because the Workflow might start and finish before the Update methods are called.